pkg/decoder.go

```diff
@@ -0,0 +1,276 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package decoder
```

Comment:
should this file be placed in `pkg/decoder/decoder.go` or perhaps better `pkg/cdclog/decoder.go`

Reply:
I prefer put it in `pkg/cdclog/decoder.go` @overvenus WDYT?